pipeline_logs/firestore_pipeline_logs_table.py

```diff
@@ -0,0 +1,37 @@
+import firebase_admin
+from core_data_modules.logging import Logger
+from firebase_admin import credentials, firestore
+
+
+log = Logger(__name__)
+
+
+class FirestorePipelinesLogsTable(object):
+
+    def __init__(self, cert):
```

Comment:
I wonder if it makes sense to put the pipeline_name and run_id in here too, so that we don't have to pass these around every time we log an event.

Reply:
updated thanks.